frontend/app/routes/$lang/_public/status/child/index.tsx

```diff
@@ -341,7 +342,7 @@ export default function StatusCheckerChild() {
                   errorMessage={errorMessages['input-radio-child-has-sin-option-0']}
                   required
                 />
-                {childHasSinState === true && <InputField id="sin" name="sin" label={t('status:child.form.sin-label')} helpMessagePrimary={t('status:child.form.sin-description')} required errorMessage={errorMessages.sin} />}
+                {childHasSinState === true && <InputSinField id="sin" name="sin" label={t('status:child.form.sin-label')} helpMessagePrimary={t('status:child.form.sin-description')} required errorMessage={errorMessages.sin} defaultValue={''} />}
```

Comment:
Don't use curly braces.

```suggestion
                {childHasSinState === true && <InputSinField id="sin" name="sin" label={t('status:child.form.sin-label')} helpMessagePrimary={t('status:child.form.sin-description')} required errorMessage={errorMessages.sin} defaultValue="" />}
```